src/routes/chat/+page.svelte

```diff
@@ -252,6 +279,18 @@
         await navigator.clipboard.writeText(txt)
     }
 
+    async function copyFile(message: string, attachment: Attachment) {
+        if (attachment.kind !== MessageAttachmentKind.Image) return
+        let result = await RaygunStoreInstance.getAttachmentRaw($conversation!.id, message, attachment.name, { size: attachment.size, type: "image/png" })
```

Comment:
no. clipboard write only supports png

Reply:
it seems that you can just use the data and treat is as a png.
e.g. i uploaded a jpeg and it allows to 
1. use a png mime
2. pasting the byte data copied into the clipboard somewhere (e.g. in discord chat) else successfully pastes its data too. it will just be png data

can also be that warp auto converts the data into png format or js `Blob` does that but i dont think that is the case